.gitignore

```diff
@@ -2,6 +2,7 @@
 *.retry
 pontus.larsson_medsci.uu.se.key
 charon_credentials.yml
-statusdb_creds.yml
+statusdb_creds_prod.yml
```

Comment:
@sylvinite shouldn't you change https://github.com/NationalGenomicsInfrastructure/irma-provision/blob/0523da869b8a9c02e5014e9d25ec7bdfb470b8d4/roles/ngi_reports/tasks/dependencies.yml#L30 and https://github.com/NationalGenomicsInfrastructure/irma-provision/blob/121c1e2c2fb1086e05994c0cb7c32c791a91db79/roles/multiqc/tasks/main.yml#L28 to read `{{ ngi_pipeline_conf }}/statusdb_creds.yml` (or something) instead of `{{ ngi_pipeline_conf }}/{{ statusdb_creds }}`. I guess it would make things easier downstreams.